pytorch_lightning/utilities/apply_func.py

```diff
@@ -35,6 +35,9 @@
     Batch = type(None)
 
 
+CPU_DEVICES = [None, "cpu", torch.device("cpu")]
```

Comment:
```suggestion
CPU_DEVICES = (None, "cpu", torch.device("cpu"))
```

Constants should not be mutable :)